2363/2/battler.rb

```diff
@@ -0,0 +1,33 @@
+require 'russian_obscenity'
+require 'terminal-table'
+
+# Class battler with stats:
+# name of battler, number of all battles,
+# all bad words for life, bad words per one
+# battle, words per one round, numbers of
+# round, words that battler said
+class Battler
+  attr_reader :name, :stats, :bad_words_per_battle
+  def initialize(*args)
```

Comment:
*args is bad practise to use, because it is absolutely not clear which params you expect there